webview-ui/src/index.css

```diff
@@ -358,3 +358,11 @@ vscode-dropdown::part(listbox) {
 input[cmdk-input]:focus {
 	outline: none;
 }
+
+/**
+ * Markdown
+ */
+
+.custom-markdown > pre {
+	background-color: transparent !important;
```

Comment:
VSCode webviews automatically adds a background color to all `pre` tags, but we don't want this when rendering code snippets without our Markdown component.